C/Curl/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Curl"
+version = v"7.66.0"
+
+# Collection of sources required to build Curl
+sources = [
+    "https://curl.haxx.se/download/curl-$(version).tar.bz2" =>
+    "6618234e0235c420a21f4cb4c2dd0badde76e6139668739085a70c4e2fe7a141"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/curl-*/
+./configure --prefix=$prefix --host=$target --build=${MACHTYPE} --disable-static
```

Comment:
I had to specify the build system to convince `configure` that this is a cross compilation :confused:

Reply:
That seems fine to me.

Reply:
Is there something you don't like about that?